test.js

```diff
@@ -25,6 +25,16 @@ test('have a compact option', t => {
 	t.is(m(1000 * 60 * 67 * 24 * 465, {compact: true}), '~1y');
 });
 
+test('limit number of units shown', t => {
+	t.is(m(1000 * 60, {unitsToShow: 0}), '~1m');
+	t.is(m(1000 * 60, {unitsToShow: 1}), '~1m');
+	t.is(m(1000 * 60 * 67, {unitsToShow: 1}), '~1h');
+	t.is(m(1000 * 60 * 67, {unitsToShow: 2}), '~1h 7m');
+	t.is(m(1000 * 60 * 67 * 24 * 465, {unitsToShow: 1}), '~1y');
+	t.is(m(1000 * 60 * 67 * 24 * 465, {unitsToShow: 2}), '~1y 154d');
+	t.is(m(1000 * 60 * 67 * 24 * 465, {unitsToShow: 3}), '~1y 154d 6h');
```

Comment:
I'd like to see some tests with this in combination with the `verbose` and `compact` option.